src/components/rangeUsePlanPage/versionsList/VersionsDropDown.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef } from 'react'
+import useSWR from 'swr'
+import * as API from '../../../constants/api'
+import { axios, getAuthHeaderConfig } from '../../../utils'
+import VersionsDropDownList from './VersionsDropDownList'
+
+const sortVersions = (a, b) => {
+  if (b.version === -1) return 1
+  if (b.version > a.version) return 1
+  if (b.version < a.version) return -1
+  return 0
+}
+
+const VersionsDropDown = ({ match, open }) => {
```

Comment:
Can we change this to `VersionsDropdown` to keep consistency with capitalization used in the rest of the app?

Reply:
Changed in this commit: c155a30

Reply:
It looks like the file is still named `VersionsDropDown.js`. Since you're on a case-insensitive filesystem, you may have to use `git mv`.

Reply:
Also, this is what is causing your tests to fail in CI.